fixtures/autofix-namespace-missing-default-deny-netpol.yml

```diff
@@ -21,5 +21,5 @@ spec:
         podSelector:           # chooses pods with type=monitoring
           matchLabels:
             type: monitoring
-    - ports: # This is comment 1
-      - port: 3452 #This is comment 2
+    - ports:
```

Comment:
What happened to this and the other comments?

Reply:
These caused tests to fail because of the bug in goyaml but since they're covered by the PreserveComments test, I took them out to make sure nothing else was causing the tests to fail. I can put them back but I think the PreserveComments test is thorough enough. What do you think?